CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.1.1 (2021-05-24)
+
+  * Support for plots rendering via [`VegaLite`](https://github.com/elixir-nx/vega_lite) ([#287](https://github.com/elixir-nx/livebook/pull/287))
+  * Deterministic section anchor links ([#288](https://github.com/elixir-nx/livebook/pull/288))
+  * Improved path selector to properly handle text editing in the middle ([#283](https://github.com/elixir-nx/livebook/pull/283))
```

Comment:
This is somehow arbitrary subset of changes, but in this case it probably makes sense to list things relevant/visible to app's end users?